src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionFetchTestResultViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+// 🦠 Corona-Warn-App
+//
+//  ExposureSubmissionFetchTestResultViewController.swift
+
+import Foundation
+import UIKit
+
+class ExposureSubmissionFetchTestResultViewController: DynamicTableViewController {
```

Comment:
I think this name might be confusing, because this view controller does not fetch the test result.  The fetch makes it sound like a network operation, but it is actually fetched in a completely different place. I kind of liked the old `Intro` name because you immediately know that this is the first view controller in this flow.